<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="baseSettingLabel">
        <property name="text">
         <string>基础设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="videoSettingLabel">
        <property name="text">
         <string>视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gifSettingLabel">
        <property name="text">
         <string>动图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imgSettingLabel">
        <property name="text">
         <string>静态</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="taskSettingLabel">
        <property name="text">
         <string>任务栏</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>430</width>
           <height>425</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="basePoint">
            <property name="text">
             <string>基础设置</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>45</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>音量</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="isAutoRun">
            <property name="text">
             <string>开机自启</string>
            </property>
           </widget>
          </item>
          <item row="10" column="4" colspan="2">
           <widget class="QPushButton" name="addGif">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="videoPoint">
            <property name="text">
             <string>视频</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QPushButton" name="addVideo">
            <property name="text">
             <string>添加文件/文件夹</string>
            </property>
           </widget>
          </item>
          <item row="10" column="6" colspan="2">
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>45</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="0">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>45</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="gifPoint">
            <property name="text">
             <string>动图</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>45</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="12" column="7">
           <spacer name="horizontalSpacer_14">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="2" colspan="6">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>260</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="imgPoint">
            <property name="text">
             <string>静态</string>
            </property>
           </widget>
          </item>
          <item row="6" column="7">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>138</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>文件</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1" colspan="3">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="7" column="3" colspan="5">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>233</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="12" column="1">
           <widget class="QSpinBox" name="intervalBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item row="6" column="5" colspan="2">
           <widget class="QLabel" name="vloumeValue">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="14" column="1" colspan="4">
           <widget class="QComboBox" name="taskBarPattern">
            <item>
             <property name="text">
              <string>启动立即美化(不检测)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>启动立即美化(检测)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>不透明(按设置的颜色来)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>毛玻璃</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>系统默认(下次一定)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>播放模式</string>
            </property>
           </widget>
          </item>
          <item row="12" column="4">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>秒</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="taskPoint">
            <property name="text">
             <string>任务栏</string>
            </property>
           </widget>
          </item>
          <item row="14" column="7">
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="isStop">
            <property name="text">
             <string>默认暂停</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2" colspan="3">
           <widget class="QSlider" name="volume">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickInterval">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>切换间隔</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>45</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3" colspan="5">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>233</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="isSimpleExit">
            <property name="text">
             <string>直接退出</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="patternBox">
            <item>
             <property name="text">
              <string>视频</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>动图</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>静态</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="3" colspan="5">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>233</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="3" colspan="5">
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="8" column="0" colspan="8">
           <layout class="QGridLayout" name="showVideos"/>
          </item>
          <item row="13" column="0" colspan="8">
           <layout class="QGridLayout" name="showImgs" columnstretch="0"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="apply">
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
